<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-81MK3MI" failures="1" tests="6" name="Default test" time="0.106" errors="0" timestamp="26 Nov 2024 13:44:10 GMT">
  <testcase classname="PCMC_All_Tests.PCMC_alltypepayments" name="@AfterMethod aftermethod" time="0.013">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;com.aventstack.extentreports.ExtentReports.flush()&amp;quot; because &amp;quot;PCMC_All_Tests.PCMC_alltypepayments.extent&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "com.aventstack.extentreports.ExtentReports.flush()" because "PCMC_All_Tests.PCMC_alltypepayments.extent" is null
	at PCMC_All_Tests.PCMC_alltypepayments.aftermethod(PCMC_alltypepayments.java:581)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:523)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:224)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:723)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- @AfterMethod aftermethod -->
  <testcase classname="PCMC_All_Tests.PCMC_alltypepayments" name="@AfterTest aftertest" time="0.0">
    <skipped/>
  </testcase> <!-- @AfterTest aftertest -->
  <testcase classname="PCMC_All_Tests.PCMC_alltypepayments" name="CashPayment" time="0.015">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;com.aventstack.extentreports.ExtentReports.createTest(String)&amp;quot; because &amp;quot;PCMC_All_Tests.PCMC_alltypepayments.extent&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "com.aventstack.extentreports.ExtentReports.createTest(String)" because "PCMC_All_Tests.PCMC_alltypepayments.extent" is null
	at PCMC_All_Tests.PCMC_alltypepayments.CashPayment(PCMC_alltypepayments.java:82)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- CashPayment -->
  <testcase classname="PCMC_All_Tests.PCMC_alltypepayments" name="chequePayment" time="0.0">
    <skipped/>
  </testcase> <!-- chequePayment -->
  <testcase classname="PCMC_All_Tests.PCMC_alltypepayments" name="cheque_fail" time="0.0">
    <skipped/>
  </testcase> <!-- cheque_fail -->
  <testcase classname="PCMC_All_Tests.PCMC_alltypepayments" name="cheque_clear" time="0.0">
    <skipped/>
  </testcase> <!-- cheque_clear -->
  <testcase classname="PCMC_All_Tests.PCMC_alltypepayments" name="Card" time="0.0">
    <skipped/>
  </testcase> <!-- Card -->
  <testcase classname="PCMC_All_Tests.PCMC_alltypepayments" name="AdvancePayment" time="0.0">
    <skipped/>
  </testcase> <!-- AdvancePayment -->
</testsuite> <!-- Default test -->
